src/ui/icons/icon-cart.svelte

```diff
@@ -0,0 +1,7 @@
+<script>
+  import Icon from "../../assets/cart.svg?raw"; // Load SVG as a raw string
+</script>
+
+<div style="color:var(--rc-color-gray-text-dark);">
```

Comment:
Here I modified the svg file to use `fill="currentColor"`.

In this way the css style can change the color of the resulting icon.
That's how fontawesome svg icons work as well.

Reply:
yeah, this is a good solution, I saw the SVG rewriting before and didn't like that very much either